WalletWasabi/BitcoinCore/P2pNode.cs

```diff
@@ -66,7 +66,7 @@ public async Task ConnectAsync(CancellationToken cancel)
 
 			if (!Node.PeerVersion.Services.HasFlag(NodeServices.Network))
 			{
-				throw new InvalidOperationException("Wasabi cannot use the local node because it does not provide blocks.");
+				throw new InvalidOperationException("We are unable to use the local node because it does not provide blocks.");
```

Comment:
I would prefer it remaining `Wasabi`, but I especially don't like the `We...` here, so the second best option would be...

```suggestion
				throw new InvalidOperationException("Unable to use the local node because it does not provide blocks.");
```